Formula/haskell-language-server.rb

```diff
@@ -63,11 +76,11 @@ def caveats
       f :: Int -> Int
     EOS
 
-    %w[ghc@8.6 ghc@8.8 ghc].each do |ghc_str|
-      ghc = Formula[ghc_str]
-
+    ghcs.each do |ghc|
       assert_match "Completed (1 file worked, 1 file failed)",
-        shell_output("PATH=#{ghc.bin}:$PATH #{bin}/haskell-language-server-#{ghc.version} #{testpath}/*.hs 2>&1", 1)
+        with_env(PATH: "#{ghc.bin}:$PATH") do
```

Comment:
Oops. This should probably be
```suggestion
        with_env(PATH: "#{ghc.bin}:#{ENV["PATH"]}") do
```
Will fix this in a bit.